From 68be2fe2021f314331e00caf55dbc53251c79c7a Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 29 Sep 2021 23:39:07 +0300
Subject: [PATCH 148/322] [FIO extras] imx8mm_evk: enable IMX_WATCHDOG

After enabling SPL by default the IMX_WATCHDOG should be enabled in
order to build u-boot successfully.

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 configs/imx8mm_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mm_evk_defconfig b/configs/imx8mm_evk_defconfig
index dcb6c320773..9ea250310e2 100644
--- a/configs/imx8mm_evk_defconfig
+++ b/configs/imx8mm_evk_defconfig
@@ -37,6 +37,7 @@ CONFIG_SPL_SEPARATE_BSS=y
 CONFIG_SPL_I2C_SUPPORT=y
 CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_SPL_WATCHDOG_SUPPORT=y
+CONFIG_IMX_WATCHDOG=y
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_SYS_PROMPT="u-boot=> "
 # CONFIG_CMD_EXPORTENV is not set
-- 
2.34.1

